spring-boot-project/spring-boot-docs/src/main/asciidoc/spring-boot-features.adoc

```diff
@@ -4377,6 +4377,29 @@ in your configuration, e.g. `spring.data.neo4j.uri=file://var/tmp/graph.db`.
 
 
 
+[[boot-features-neo4j-ogm-native-types]]
+==== Using native types
+Neo4j-OGM can map some types, like `java.time.*` to `String` based properties or
+use one of the various native types that Neo4j provide
+(Please refer to the https://neo4j.com/docs/developer-manual/3.4/drivers/cypher-values/#driver-neo4j-type-system[Cypher type system] for a full list).
+For backwards compatibility reasons the default for Neo4j-OGM is using a `String` based representation.
+To change that behaviour, please add one of `org.neo4j:neo4j-ogm-bolt-native-types` or `org.neo4j:neo4j-ogm-embedded-native-types`
+to the dependencies of your application
+and use the following property in your Spring Boot configuration:
+
+[source,properties,indent=0]
+----
+	spring.data.neo4j.use-native-types=true
+----
+
+With this setting you can use Java 8 temporal types without any conversion necessary in your domain classes.
+
+[NOTE]
+====
+This feature is only available if Neo4j-OGM uses Bolt or embedded transport mode.
+====
```

Comment:
I feel that part of the description here would be better suited in the Neo4j documentation. We could have a link that explains this feature rather than explaining it here again.

Reply:
I'm gonna change that including the hardcoded version number. There's a chapter on that on the 3.2 docs, it's just not uploaded to our docs server. Good point.